src/lib.rs

```diff
@@ -1,5 +1,9 @@
 // SPDX-License-Identifier: Apache-2.0
 
+//! ![test](https://github.com/enarx/iocuddle/workflows/test/badge.svg)
+//! ![lint](https://github.com/enarx/iocuddle/workflows/lint/badge.svg)
+//! ![rust v1.36+](https://img.shields.io/badge/rustc-v1.36%2B-blue.svg)
```

Comment:
I think we want to follow the badges workflow outlined here: https://github.com/livioribeiro/cargo-readme

Reply:
yes please

Reply:
Hmm, for the `[rust v1.36+]` badge, we would have to use a `README.tpl` template, because it's not a Cargo.toml standard.

Reply:
you might want to have https://github.com/enarx/iocuddle/pull/7 first